api/openapi.gen.yaml

```diff
@@ -156,8 +156,7 @@ components:
   schemas:
     v1.Account:
       properties:
-        account_number:
-          type: string
+        account_number: {}
```

Comment:
This is not right, we want to either expose the whole String/Valid struct or preferably make our own marhsaler because JSON does support null values natively. Let me explain in the review comment.